src/StoreApi/Routes/V1/Checkout.php

```diff
@@ -92,11 +92,11 @@ public function get_args() {
 							],
 						],
 					],
-					$this->schema->get_endpoint_args_for_item_schema( \WP_REST_Server::CREATABLE )
```

Comment:
What is this change for?

Reply:
I don't remember exactly. Most definitely a mistake since it does not relate to the scope of this task. I'll revert it, thanks for catching it.
Btw, it was redundant since it is the default value, and we already state the method type on line 74, guess it got flagged by the IDE.